src/commands/dev/index.ts

```diff
@@ -503,7 +503,20 @@ export default class Dev extends BaseCommand {
       build_args.push('--build-arg', arg);
     }
 
-    await DockerComposeUtils.dockerCompose(['-f', compose_file, '-p', project_name, 'build', ...build_args], { stdio: 'inherit' });
+    const cmd = DockerComposeUtils.dockerCompose(['-f', compose_file, '-p', project_name, 'build', ...build_args], { stdin: 'inherit', stdout: 'inherit', stderr: 'pipe' });
+
+    let stderr_message = '';
+    try {
+      cmd.stderr?.on('data', (bytes) => {
+        stderr_message = (bytes.toString() || '').trim();
+      });
```

Comment:
I'm pretty sure this information is in the exception object returned by execa - I believe `e.stderr`is available so we don't need to set the string this way. You can run `console.log(e)` to quickly see the various keys available - but I think that would be the same information that's being captured here

Reply:
removed  on-data function